.github/workflows/extract-translation-source-files.yml

```
@@ -43,6 +43,7 @@ jobs:
           - credentials
           - DoneXBlock
           - RecommenderXBlock
+          - xblock-adventure
```

@shadinaif this PR is now conflicting with the recently merged Python module names PR #353. Please rebase and use the edit below:

```suggestion
          - repo_name: xblock-adventure
            python_module_name: adventure
```